tests/library/browsercontext-fetch.spec.ts

```diff
@@ -122,22 +122,40 @@ it('should add session cookies to request', async ({ context, server }) => {
 });
 
 for (const method of ['fetch', 'delete', 'get', 'head', 'patch', 'post', 'put'] as const) {
-  it(`${method} should support queryParams`, async ({ context, server }) => {
-    const url = new URL(server.EMPTY_PAGE);
-    url.searchParams.set('p1', 'v1');
-    url.searchParams.set('парам2', 'знач2');
-    const [request] = await Promise.all([
-      server.waitForRequest(url.pathname + url.search),
```

Comment:
This was checking that the parameters are actually sent to the server, whereas now the test only checks them on the client end. Let's keep the server-side check too.